dart.html.markdown

```diff
@@ -2,28 +2,72 @@
 language: dart
 filename: learndart.dart
 contributors:
-    - ["Joao Pedrosa", "https://github.com/jpedrosa/"]
+  - ["Joao Pedrosa", "https://github.com/jpedrosa/"]
+  - ["Vince Ramces Oliveros", "https://github.com/ram231"]
 ---
 
-Dart is a newcomer into the realm of programming languages.
-It borrows a lot from other mainstream languages, having as a goal not to deviate too much from
-its JavaScript sibling. Like JavaScript, Dart aims for great browser integration.
+**Dart** is a single threaded, general puprose programming languages.
+It borrows a lot from other mainstream languages.
+It supports Streams, Futures(known as Promises in JavaScript), Generics, First-class functions(closures) and static type checking.
+Dart can run in any platform such as Web, CLI,Desktop, Mobile and IoT devices.
```

Comment:
add space after "CLI,"
Also, maybe we could add something about running on a VM or native (ARM and X86)